articles/01-DownloadingTheXRTK.md

```diff
@@ -2,56 +2,63 @@
 
 The Mixed Reality Toolkit provides many ways for users and developers to get access to the Mixed Reality Toolkit and the various extensions/platforms it supports.  These methods are tuned to use the common patterns most familiar to Unity developers, these include:
 
-* [Register UPM Server in Project Settings](#register-upm-server-in-project-settings) - Register the Mixed Reality Toolkit in the Unity Editor in the project settings.
-* [Package Seed Installation](#package-seed-installation) - Automatically register the Mixed Reality Toolkit's upm server and install selected packages using the package seed found in the [release assets dropdown](https://github.com/XRTK/XRTK-Core/releases).
+* [Automatic UPM installation](#automatic-upm-installation) - Automatically register the Mixed Reality Toolkit's upm server and install selected packages using the package seed found in the [release assets dropdown](https://github.com/XRTK/XRTK-Core/releases).
 * [Manual UPM installation](#manual-upm-installation) - Register the Mixed Reality Toolkit upm server in your manifest and manually download modules al la carte.
 * [UPM via GitHub](https://docs.unity3d.com/Manual/upm-ui-giturl.html) - The Unity Package Manager can load a package from GitHub using the git url.
   > **Note:** To utilize this method, use the `upm` branch for each respective project. For Example:
   >
   > `https://github.com/XRTK/XRTK-Core.git#upm`
-* [GIT Submodules](#git-submodules) - For advanced developers who want to use the Mixed Reality Toolkit in their live project and contribute directly back to the project in real time.
+* [GIT Submodules](#git-submodules) - For advanced developers who want to use the Mixed Reality Toolkit in their live project and contribute direcly back to the project in real time.
 
 In this article, we will walk through each approach to get you up and running, starting with the simplest first.
 
-Our preferred deployment approach is to fully utilize [Unity's native package manager](https://docs.unity3d.com/Manual/upm-ui.html) to incorporate the Mixed Reality Toolkit in your solution, akin to the other modules Unity provides out of the box.  This is the quickest and safest way to get Mixed Reality Toolkit in your solution and is automatically updated as new releases are published.
```

Comment:
Why remove this?

Reply:
Because it's been replaced by an entire section on using the Unity package manager

Reply:
Let's keep it removed. It's just boilerplate text. There is only one way to get the XRTK into the project and that is the UPM way.